internal/config/globalConfig.go

```diff
@@ -0,0 +1,69 @@
+//
+// Licensed to Apache Software Foundation (ASF) under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Apache Software Foundation (ASF) licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package config
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	"github.com/apache/skywalking-infra-e2e/internal/logger"
+
+	"gopkg.in/yaml.v2"
+
+	"github.com/apache/skywalking-infra-e2e/internal/util"
+)
+
+// GlobalE2EConfig store E2EConfig which can be used globally.
+type GlobalE2EConfig struct {
+	Ready     bool
+	E2EConfig E2EConfig
+}
+
+var GlobalConfig GlobalE2EConfig
+
+func ReadGlobalConfigFile(configFilePath string) error {
+	if GlobalConfig.Ready {
+		logger.Log.Info("e2e config has been initialized")
+		return nil
+	}
+
+	e2eFile := configFilePath
+	if util.PathExist(e2eFile) {
```

Comment:
Return `fmt.Errorf("e2e config file %s not exist", e2eFile)` early `if !util.PathExist`, reducing the blocks